embark-ui/src/components/Console.js

```diff
@@ -19,7 +19,21 @@ class Console extends Component {
   constructor(props) {
     super(props);
     this.typeahead = React.createRef();
-    this.state = {value: '', isLoading: true, options: [], activeTab: EMBARK_PROCESS_NAME, historyIndex: DEFAULT_INDEX};
+    this.state = {value: '', isLoading: true, options: [], activeTab: EMBARK_PROCESS_NAME, suggestions: []};
+  }
+
+  shouldUpdateSuggestions(prevProps) {
+    return !_.isEqual(prevProps.commandHistory, this.props.commandHistory) ||
+      !_.isEqual(prevProps.commandSuggestions, this.props.commandSuggestions) ||
```

Comment:
We have a library for deepEqual. No need to install lodash just for that.

Reply:
updated!